task/redhat-codeready-dependency-analysis/0.2/redhat-codeready-dependency-analysis.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+
+metadata:
+  name: redhat-codeready-dependency-analysis
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.19.0"
+    tekton.dev/tags: Security, Vulnenrability, CVE
+    tekton.dev/displayName: "RedHat CodeReady Dependency Analysis"
+
+spec:
+  description: >-
+    The RedHat CodeReady Dependency Analysis task is an interface between Tekton and Red Hat CodeReady Dependency Analytics platform.
+    It provides vulnerability and compliance analysis for your Python applications dependencies, along with recommendations to address security vulnerabilities and licensing issues.
+
+  workspaces:
+    - name: output
+      description: Volume backing this workspace is used for input/output of the task.
+
+  params:
+    - name: language
+      description: String value to donate a language; 1 = Golang, 2 = NPM, 3 = Python, 4 = Java.
```

Comment:
i would prefer a string like `golang` or `python` instead of a number as later in a if what we see is a number, then we will have to mentally map it back to a runtime/language